.env.develop

```diff
@@ -1 +1 @@
-REACT_APP_API_URL = 'http://your-api-url'
+REACT_APP_API_URL = 'https://api.explorer.nervos.org'
```

Comment:
Sensitive information should not be uploaded to open repository.

Reply:
I don't think this is sensitive information, as any one can easily get this address from explorer frontend website.

But should we have production endpoint for `.env.develop` is another story. We should look at our doc and see if we've described that clearly.

Reply:
OK. We didn't describe clearly on our document, so it's my negligence.

Reply:
Is the consensus to leave this the way it is for now?